system/settings/settings.xml

```diff
@@ -405,6 +405,32 @@
         </setting>
       </group>
       <group id="2">
+        <setting id="videoplayer.seeksteps" type="list[integer]" label="13556" help="37042">
+          <level>2</level>
+          <default>-1800,-900,-600,-300,-180,-60,-30,-15,-7,7,15,30,60,180,300,600,900,1800</default>
```

Comment:
as mentioned on IRC I'd not enable +7 by default - it's a bit low for a seek forward. -7 is fine for jumps back though